wiki/Gameplay/Accuracy/en.md

```diff
@@ -0,0 +1,5 @@
+# Accuracy
+
+Accuracy defines how accurately a player is hitting [hit objects](/wiki/Hit_Objects) on a [beatmap](/wiki/Beatmaps). This value varies from from 0 to 100%, and fluctuates up and down while a beatmap is being played. This happens because accuracy only takes objects already played into account. Accuracy can be seen in the top-right corner while watching a replay or playing a beatmap.
```

Comment:
I'm still uncertain about this paragraph. I think the second and third sentences should be removed, but I'm going to wait to see if someone else has anything to comment about this stub.

Reply:
ah, i agree. this only applies to osu! standard